### DISCUSSION TITLE
My friends can't join Host Multiplayer Server

### DISCUSSION THREAD

**Discussion by Quexixd** [OpenRCT2/OpenRCT2#16404] [Q&A]
### Operating System

Windows 11, 64 Bits

### OpenRCT2 build

OpenRCT2, v0.3.5.1 (61c67af) provided by GitHub

### Describe the issue

My friends can't joint my Multiplayer Server i searched very much didn't found a solution. I allready Port forwarded and did the settings in the windows firewall

### Area(s) with issue?

This bug is specific to multiplayer

### Steps to reproduce

1.Downloaded
2. Port forwared
3.Disabled the Port in Windows Firewall

**Comment by janisozaur** [comment]
Verify you can connect from a client in local network, provide logs, verify port is open from external network (e.g. GRC shieldsup)

**Reply by Quexixd** [reply]
My Port showes closed, but my Minecraft port also showes closed but works.
Where can i find the logs?Tested in local network and worked.

**Reply by janisozaur** [reply]
It won't work for OpenRCT2 until the ports are open. If you can connect from local network, but not from external service, the issue of isolated to your network configuration
